Lib/unittest/mock.py

```diff
@@ -1651,7 +1651,8 @@ def patch(
     "as"; very useful if `patch` is creating a mock object for you.
 
     `patch` takes arbitrary keyword arguments. These will be passed to
-    the `Mock` (or `new_callable`) on construction.
+    to construct a `AsyncMock` if the patched object is an async function,
+    to `MagicMock` otherwise or to `new_callable` if specified.
```

Comment:
Is there any way we could not have this text duplicated between mock.py and mock.rst?

Reply:
I am new to Python docs, but as far as I looked around there is a lot of duplication between .rst and .py in many modules, will need to ask around about if there is another way.

Reply:
It makes this a more substantial PR, but Sphinx autodoc should be able to pull in the .py's docstrings. Not sure if autodoc is enabled for the CPython docs, that would be some more digging on your part :-)

Reply:
Will take a look into it, but it looks like all modules actually repeats docs in `Doc` and in `Lib`, will ask around why.

Reply:
How did the asking around go?